sdk/purview/purview-workflow-rest/README.md

```diff
@@ -0,0 +1,140 @@
+
+- Manage workflows
+- Submit user requests and monitor workflow runs
+- View and respond to workflow tasks
+
+**For more details about how to use workflow, please refer to the [service documentation][product_documentation]**
+
+## Getting started
+
+### Currently supported environments
+
+- Node.js version 14.x.x or higher
+
+### Prerequisites
+
+- You must have an [Azure subscription][azure_subscription] and a [Purview resource][purview_resource] to use this package.
+
+### Create and authenticate a `PurviewWorkflowClient`
+
+Since the Workflow service uses an Azure Active Directory (AAD) bearer token for authentication and identification, an email address should be encoded into the token to allow for notification when using Workflow. It is recommended that the [Azure Identity][azure_identity] library be used  with a the [UsernamePasswordCredential][username_password_credential]. Before using the [Azure Identity][azure_identity] library with Workflow, [an application][app_registration] should be registered and used for the clientId passed to the [UsernamePasswordCredential][username_password_credential].
+Set the values of the client ID, tenant ID, username and password as environment variables:
+AZURE_CLIENT_ID, AZURE_TENANT_ID, USERNAME, PASSWORD
+
+```typescript
+import PurviewWorkflow from "@azure-rest/purview-workflow";
+import { UsernamePasswordCredential } from "@azure/identity";
+const client = PurviewWorkflow(
+  "https://<account-name>.purview.azure.com",
+  new UsernamePasswordCredential(
+        <AZURE_TENANT_ID>,
+        <AZURE_CLIENT_ID>,
+        <USERNAME>,
+        <PASSWORD>
+      )
+  );
+```
+
+## Examples
+
+The following section provides several code snippets covering some of the most common scenarios, including:
+
+- [Submit User Requests](#submit-user-requests)
+- [Approve Workflow Task](#approve-workflow-task)
+
+### Submit user requests
+
+```typescript
+import createPurviewWorkflowClient, {
+  SubmitUserRequestsParameters
+} from "@azure-rest/purview-workflow";
+import { UsernamePasswordCredential } from "@azure/identity";
+
+async function userRequestsSubmit() {
+  const endpoint = <ENDPOINT>;
+  const tenantId = <TENANTID>;
+  const clientId = <CLIENTID>;
+  const username = <USERNAME>;
+  const password = <PASSWORD>;
+  const credential = new UsernamePasswordCredential(tenantId , clientId, username, password);
+  const client = createPurviewWorkflowClient(endpoint, credential);
+  const options: SubmitUserRequestsParameters = {
+    body: {
+      comment: "Thanks!",
+      operations: [
+        {
+          type: "CreateTerm",
+          payload: {
+            glossaryTerm: {
+              name: "term",
+              anchor: { glossaryGuid: "20031e20-b4df-4a66-a61d-1b0716f3fa48" },
+              nickName: "term",
+              status: "Approved"
+            }
+          }
+        }
+      ]
+    }
+  };
+  const result = await client.path("/userrequests").post(options);
```

Comment:
Can you us `isUnexpected` here to check that the result status code is what you expect, otherwiser throw error info